Slim/Player/SqueezePlay.pm

```diff
@@ -89,6 +91,7 @@ my %CapabilitiesMap = (
 	Spdirect                => 'spDirectHandlers',
 	Proxy                   => 'proxyAddress',
 	CanHTTPS                => '_canHTTPS',
+	Balance					=> 'balance',
```

Comment:
Whitespace ;-)

Reply:
I need to have at least one of these :-)